tests/unit/test_response.py

```diff
@@ -46,6 +46,13 @@ def test_streaming_wrapper_validates_content_length(self):
         stream = response.StreamingBody(body, content_length=10)
         self.assertEqual(stream.read(), b'1234567890')
 
+    def test_streaming_wrapper_generates_line_streams(self):
+        body = six.BytesIO(b'1234567890\n1234567890')
+        stream = response.StreamingBody(body, content_length=21)
+        lines = stream.readlines()
```

Comment:
- Please do not use the methods `stream.readline()` and `stream.readlines()` for iterate over the stream/file:
```python
# Bad
while True:
    line = your_stream.readline()
    ...

for line in your_stream.readlines():
    ...


# Better
for line in your_stream:
    ...
```